app/views/ile_query_mucr_response.scala.html

```diff
@@ -14,73 +14,91 @@
  * limitations under the License.
  *@
 
-@import java.time.format.DateTimeFormatter
-@import models.viewmodels.notificationspage.NotificationsPageSingleElement
-@import views.IleCodeMapper
-@import views.Title
-@import views.Title.NO_SECTION
-@import views.ViewDates
-@import models.notifications.queries.MucrInfo
+@import components.gds.gds_main_template
+@import components.ilequery.{response_buttons, response_parent, response_previous_movements, response_associated_consignments}
+@import models.notifications.queries.{MucrInfo, UcrInfo}
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import views.components.BackButton
+@import views.{IleCodeMapper, Title, TransportMapper}
 
-@this(main_template: views.html.templates.main_template)
+@this(
+        ileCodeMapper: IleCodeMapper,
+        transportMapper: TransportMapper,
+        govukLayout: gds_main_template,
+        parentSection: response_parent,
+        previousMovementsSection: response_previous_movements,
+        associatedConsignmentsSection: response_associated_consignments,
+        responseButtons: response_buttons,
```

Comment:
Nit: IMHO it would be better to use 'result' instead of 'response'. To me response suggests the payload itself.
Might be also worth asking if it would be better to align views names with pages titles?